src/domains/access.ml

```diff
@@ -333,8 +333,16 @@ struct
       let pretty = pretty
     end
     )
+
+  let conf (conf, _, _, _, _) = conf
+end
+module AS =
+struct
+  include SetDomain.Make (A)
+
+  let max_conf accs =
+    accs |> elements |> List.map A.conf |> List.max
```

Comment:
I get a compile error here on my machine. Going to `(List.max ~cmp:Int.compare)` fixes it.

Reply:
That's weird, there was a `List.max` in this file before this PR as well. Do you have an outdated version of batteries or something? Our lower bound is 3.4.0 and that's also locked and tested by the CI, where it works.

Reply:
No, I have the most recent batteries. The same fix was also needed in the intDomain.ml at some point. Maybe it has to do with using it through `|>`?

Reply:
That fix was exactly related to batteries 3.4.0.

Reply:
Ahh, the typechecking of `|>` was specialized in OCaml 4.13: https://github.com/ocaml/ocaml/pull/10081.